Signal/test/util/UtilTest.m

```diff
@@ -3,12 +3,25 @@
 //
 
 #import "UtilTest.h"
+#import "DateUtil.h"
 #import "TestUtil.h"
 #import <SignalMessaging/NSString+OWS.h>
 #import <SignalServiceKit/NSDate+OWS.h>
 #import <SignalServiceKit/NSObject+OWS.h>
 #import <SignalServiceKit/NSString+SSK.h>
 
+@interface DateUtil (Test)
+
++ (BOOL)dateIsOlderThanToday:(NSDate *)date now:(NSDate *)now;
```

Comment:
These are "private" methods of `DateUtil` let us specify `now` which lets us make these tests deterministic and safe.